### PR TITLE
Add ANCM to runtime site extension

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

**Comment by jkotalik** [comment]
What is the verification status on this? Have you tried Inproc and out of proc on antares?

**Comment by pakrym** [comment]
Nopes, no verification yet. Waiting for IISIntegration build to go through

**Comment by pakrym** [comment]
This is working and ready for review.

File: build/repo.targets [hunk 1]
```diff
@@ -13,6 +13,7 @@
       <SiteExtensionOutputDirectory>$(RepositoryRoot)artifacts\build</SiteExtensionOutputDirectory>
       <TestProjectDirectory>$(RepositoryRoot)\test\Microsoft.AspNetCore.AzureAppServices.FunctionalTests\</TestProjectDirectory>
       <SiteExtensionFeed Condition="$(SiteExtensionFeed) == ''">https://dotnet.myget.org/F/aspnetcore-dev/</SiteExtensionFeed>
+      <SiteExtensionNETCoreAppPackageVersion>$(MicrosoftNETCoreApp22PackageVersion)</SiteExtensionNETCoreAppPackageVersion>
```
Comment (jkotalik) [inline build/repo.targets]: Need to retarget for 2.2

File: extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj [hunk 2]
```diff
@@ -10,14 +10,26 @@
...
   <ItemGroup>
     <Content Include="applicationHost.xdt" />
+    <Content Include="scmApplicationHost.xdt" />
     <Content Include="install.cmd" />
     <Content Include="$(OutputPath)\Microsoft.Web.Xdt.Extensions.dll" PackagePath="content" />
 
     <Content Include="$(DotnetHomeDirectory)\$(RuntimeArch)\**\*.*" Condition="$(DotnetHomeDirectory) != ''" PackagePath="content\%(RecursiveDir)%(FileName)%(Extension)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(RuntimeArch)' == 'x86'">
```
Comment (jkotalik) [inline extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj]: nit: Why not RuntimeArchitecture?

File: extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj [hunk 3]
```diff
@@ -10,14 +10,26 @@
...
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(RuntimeArch)' == 'x64'">
+    <Content Include="$(AspNetCoreModuleX64Location)" PackagePath="content\ancm" />
+    <Content Include="$(OutOfProcessRequestHandlerX64Location)" PackagePath="content\ancm\$(AspNetCoreModuleOutOfProcessVersion)" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.Web.Xdt.Extensions\Microsoft.Web.Xdt.Extensions.csproj" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.AspNetCore.AspNetCoreModuleV2" Version="$(MicrosoftAspNetCoreAspNetCoreModuleV2PackageVersion)" />
```
Comment (jkotalik) [inline extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj]: Do we care about V1?

**Review by jkotalik — APPROVED** [review]

File: extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj (hunk 3 above)
Comment (pakrym) [reply extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj]: Not in this PR.

File: extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj (hunk 2 above)
Comment (pakrym) [reply extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj]: RuntimeArchitecture might affect other msbuild targets and we don't want that.

File: build/repo.targets (hunk 1 above)
Comment (pakrym) [reply build/repo.targets]: Retarget what?

Comment (jkotalik) [reply build/repo.targets]: The branch to target 2.2. Sorry that wasn't clear.

**Pull request merged**